main.py

```diff
@@ -0,0 +1,41 @@
+import pandas as pd
+import glob
+import re
+import os
+# Ścieżka do folderu, w którym znajdują się pliki

```

Comment:
No need for obvious comments. Better keep them in english so more people can benefit form them.